src/script/util/messageRenderer.ts

```diff
@@ -153,7 +177,7 @@ export const renderMessage = (message: string, selfId: QualifiedId | null, menti
     if (!isWireDeepLink && !['autolink', 'linkify'].includes(link.markup)) {
       const title = link.attrGet('title');
       if (title) {
-        link.attrSet('title', title);
+        link.attrSet('title', removeMentionsHashes(title));
       }
       link.attrSet('href', href);
       if (nextToken?.type === 'text') {
```

Comment:
```suggestion
```
remove link.addrSet and the following if (it does nothing), but GitHub does not allow to select all unchanged lines